src/main/java/stream/flarebot/flarebot/util/ReportManager.java

```diff
@@ -1,25 +1,28 @@
 package stream.flarebot.flarebot.util;
 
-import org.joda.time.DateTime;
 import stream.flarebot.flarebot.FlareBot;
 import stream.flarebot.flarebot.objects.Report;
 import stream.flarebot.flarebot.objects.ReportStatus;
 
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Timestamp;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
 
 public final class ReportManager {
 
     private ReportManager(){
     }
 
-    private final List<Report> reportsToSave = new ArrayList<>();
+    Map<String, Set<Report>> reports = new HashMap<>();
 
     public List<Report> getGuildReports(String guildID) {
-        List<Report> reports = new ArrayList<>();
+        Set<Report> reports;
+        if(this.reports.containsKey(guildID)){
+            reports = this.reports.get(guildID);
+        } else {
+            reports = new HashSet<>();
+        }
         try {
```

Comment:
Oh sorry, this is actually about e code below due to the mysql but also do getOrDefault